src/config.rs

```diff
@@ -0,0 +1,131 @@
+use anyhow::Result;
+use serde::Deserialize;
+use std::collections::HashMap;
+use std::fs;
+use std::path::Path;
+
+/// Represents a single conversion configuration.
+#[derive(Deserialize, Debug)]
+struct Conversion {
+    topic: String,
+    frame_id: String,
+    ros_type: String,
+    converter: String,
```

Comment:
This is probably fine for now, but my intuition is that these parameters are redundant in some way.

Ideally we would have a registry of converters and each converter would know it's input ros_type.

If we only have a single converter for a given type, then we can infer it from the ros_type.

Alternatively, if you've specified a converter explicitly, then we should already know what the ros_type needs to be for it.

Reply:
Yeah, I agree. I ended up dropping the `converter` parameter in https://github.com/rerun-io/rerun-ros/pull/12 when I realized they were redundant.